src/lib.rs

```diff
@@ -391,6 +395,32 @@ impl Deref for AlgorithmIdentifier {
     }
 }
 
+/// A timestamp, tracking the number of non-leap seconds since the start of 1970
```

Comment:
I think it would be good to mention the term "Unix epoch" in the API docs as that's well understood to be `1970-01-01T00:00:00+00:00` (timezone matters)